### ISSUE TITLE
Jasmine tests logging an error in the console: cannot read property 'match' of undefined

### ISSUE THREAD

**Issue by cjsauer** [Sanjo/meteor-jasmine#228]
Not sure if this error is important or not, but it is a tad annoying. I get this error in the chrome console as soon as I start up the server by running `meteor`. It doesn't show in the Karma chrome instance, only in the normal chrome browser when I navigate to localhost:3000. It doesn't prevent the test from passing; it still runs normally. 

```
Exception in setTimeout callback: TypeError: Cannot read property 'match' of undefined
    at translateUrl (http://localhost:3000/packages/ddp.js?d1840d3ba04c65ffade261f362e26699b7509706:2780:24)
    at toSockjsUrl (http://localhost:3000/packages/ddp.js?d1840d3ba04c65ffade261f362e26699b7509706:2830:10)
    at _.extend._launchConnection (http://localhost:3000/packages/ddp.js?d1840d3ba04c65ffade261f362e26699b7509706:2708:30)
    at new LivedataTest.ClientStream (http://localhost:3000/packages/ddp.js?d1840d3ba04c65ffade261f362e26699b7509706:2582:8)
    at new Connection (http://localhost:3000/packages/ddp.js?d1840d3ba04c65ffade261f362e26699b7509706:3522:20)
    at Object.DDP.connect (http://localhost:3000/packages/ddp.js?d1840d3ba04c65ffade261f362e26699b7509706:5090:13)
    at http://localhost:3000/packages/sanjo_jasmine.js?46996959cf65cb76c4b8a9c4a65eaf702d790e8a:4236:35
    at _.extend.withValue (http://localhost:3000/packages/meteor.js?43b7958c1598803e94014f27f5f622b0bddc0aaf:955:17)
    at withoutInvocation (http://localhost:3000/packages/meteor.js?43b7958c1598803e94014f27f5f622b0bddc0aaf:435:45)
    at Meteor.bindEnvironment (http://localhost:3000/packages/meteor.js?43b7958c1598803e94014f27f5f622b0bddc0aaf:983:22)
```

I've created a matching issue on my Meteor Fresh starter template: https://github.com/cjsauer/meteor-fresh/issues/2

**Comment by raix** [comment]
I'm also running into this:

It fails at: https://github.com/Sanjo/meteor-jasmine/blob/master/src/client/integration/ClientIntegrationTestFramework.js#L106

`PARENT_URL` is set by the test env: https://github.com/Sanjo/meteor-jasmine/blob/master/test-app/start_integration_mirror.sh#L9

I changed: https://github.com/Sanjo/meteor-jasmine/blob/master/src/server/lib/mirror-info.js#L8-L12

to:
```js
    if (isTestPackagesMode()) {
      // If in test mode use the PARENT_URL set by the test env script
      mirrorInfo.parentUrl = process.env.PARENT_URL
    } else {
      mirrorInfo.parentUrl = process.env.ROOT_URL
    }
```

@Sanjo please confirm if this fix is valid

--

Notes: https://github.com/Sanjo/meteor-jasmine/blob/master/src/client/integration/ClientIntegrationTestFramework.js#L101 could use `Meteor.defer` instead?

**Comment by Sanjo** [comment]
Fixed in 0.15.3.

@raix Your fix is not valid ;-) :-)  I now use Meteor.defer.

**Comment by raix** [comment]
thanks @Sanjo :)
